### PR TITLE
Docs, javadocs, version bumps, ...

### PR DESCRIPTION
**Pull request by whiskeysierra**

### CODE REVIEW

File: riptide-core/src/main/java/org/zalando/riptide/Completion.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionException;
 
+// TODO move to riptide-capture
```
Comment (lukasniemeier-zalando) [inline riptide-core/src/main/java/org/zalando/riptide/Completion.java]: Is this a `TODO` for this PR?

File: riptide-core/src/main/java/org/zalando/riptide/Navigator.java [hunk 2]
```diff
@@ -10,6 +10,13 @@
 import java.util.Map;
 import java.util.Optional;
 
+/**
+ * A Navigator chooses among the {@link Binding bindings} of a {@link RoutingTree routing tree\. The act of traversing
```
Comment (lukasniemeier-zalando) [inline riptide-core/src/main/java/org/zalando/riptide/Navigator.java]: `{@link RoutingTree routing tree\` --> `{@link RoutingTree routing tree}`

File: riptide-core/src/main/java/org/zalando/riptide/Rest.java [hunk 3]
```diff
@@ -4,13 +4,28 @@
...
+ *
+ * <pre>{@code http.get("/users")
+ *     .queryParam("active", "true")
+ *     .accept(APPLICATION_JSON)
+ *     .dispatch(..)}</pre>
+ *
+ * @see RestTemplate
+ * @see AsyncRestTemplate
+ */
+// TODO rename to Http?
```
Comment (lukasniemeier-zalando) [inline riptide-core/src/main/java/org/zalando/riptide/Rest.java]: I assume this `TODO` is not for this PR, but I like it.

File: riptide-core/src/main/java/org/zalando/riptide/Completion.java (hunk 1 above)
Comment (whiskeysierra) [inline riptide-core/src/main/java/org/zalando/riptide/Completion.java]: Follow-up, already prepared.

File: riptide-core/src/main/java/org/zalando/riptide/Rest.java (hunk 3 above)
Comment (whiskeysierra) [inline riptide-core/src/main/java/org/zalando/riptide/Rest.java]: Another follow-up, already prepared.

File: riptide-core/src/main/java/org/zalando/riptide/Navigator.java (hunk 2 above)
Comment (whiskeysierra) [inline riptide-core/src/main/java/org/zalando/riptide/Navigator.java]: Fixed.

**Comment by lukasniemeier-zalando** [comment]
:+1:

**Pull request merged**